libs/widget-lib/src/cowSwapWidget.ts

```diff
@@ -137,3 +145,29 @@ function applyDynamicHeight(iframe: HTMLIFrameElement, defaultHeight = DEFAULT_H
     iframe.style.height = event.data.height ? `${event.data.height + HEIGHT_THRESHOLD}px` : defaultHeight
   })
 }
+
+/**
+ * Subscribes to the cow events
+ *
+ * @param events - subscription to different events
+ */
+function subscribeToCoWEvents(events?: CowEventSubscriptions): void {
+  if (!events) {
+    return
+  }
+
+  const eventEmitter = new CowEventEmitterImpl()
+
+  window.addEventListener('message', (event) => {
```

Comment:
Is there a type for this `event`?

Reply:
No, it's kind of a standard https://developer.mozilla.org/en-US/docs/Web/API/Window/message_event